packages/node/src/builder.ts

```diff
@@ -3,7 +3,7 @@ import { optimizeCss } from '@pandacss/core'
 import { ConfigNotFoundError } from '@pandacss/error'
 import { logger } from '@pandacss/logger'
 import { existsSync, statSync } from 'fs'
-import { resolve } from 'pathe'
+import { normalize, resolve } from 'pathe'
```

Comment:
@segunadebayo (cc: @astahmer): Sorry to have communicated poorly previously. There's still the same issue with Windows HMR in the new release (`v0.27.1`) because `pathe.normalize` doesn't swap forward slash to backslash in Windows paths like `path.normalize` does.

That results in paths that still look like `C:/Users/test/Documents/panda-css-app/app` (forward slash) instead of `C:\\Users\\test\\Documents\\panda-css-app\\app` (backslash). Since `webpack` will reject the forward slash version, we still get the same HMR issue in Windows.

If we use the `node:path` version of `normalize` we should get the behavior we're looking. To my knowledge, in the parts of the code that deal with generating descriptions of `postcss` (and by extension `webpack`) dependencies, we don't need to / want to use `pathe` as it leads to the wrong path normalization.

There may be reasons you want/need `pathe` in other parts of the code (maybe the CLI generator?) but at the boundary with `postcss`/`webpack` it actually hurts us.

Sorry that wasn't clear from my previous comments / commits!

Reply:
@mewdriller I agree.

Switched `pathe` to `node:path` here #2012